src/xdocs/config_blocks.xml

```diff
@@ -277,6 +264,18 @@ switch (a) {
       <subsection name="Examples" id="EmptyBlock_Examples">
         <p> To configure the check: </p>
         <source>
+&lt;module name=&quot;EmptyBlock&quot;&gt;
+  &lt;property name=&quot;tokens&quot; value=&quot;LITERAL_DEFAULT&quot;/&gt;
```

Comment:
Please move it below default config/example

Reply:
Done